README.md

```diff
@@ -1,57 +1,96 @@
-The env0 Extension for Visual Studio Code (VS Code) lets you interact with your env0 project form your IDE!
+## 🌟 Features
 
-## Features
+### 🌲 View Environments
+- Easily access all environments related to the current branch you are working on.
 
-* automatically filters environments that are associated with the branch you're currently working on
-* deploy, destroy, cancel, abort environments
-* get notifications about environments' changing statuses, and when waiting for approval
-* get live deployment log field in your VS Code terminal
+### 🚀 Execute Actions
+- Perform various actions such as `Deploy`, `Destroy`, `Cancel`, and `Abort` without leaving your IDE.
 
-## Requirements
+### 📜 Real-time Logs
+- Once an environment is selected, view the most recent deployment logs in the env0 output channel.
 
-You need to have a pre-existing env0 organization, and provide an api-key in your *DEBUGGING* VS Code's settings:  
-go to settings (default shortcut CMD+,)  
-and set both - api key id, and api secret key  
-and you're done
+### 🛎 Notifications
+- Stay informed with real-time notifications regarding changing environment statuses and pending approvals.
 
-## Development Workflow
-1. clone this repo
-2. `pnpm install`
-3. go to the debug panel on the left side (CMD+SHFT+D)
-4. on the `RUN AND DEBUG` select-box, choose `Run Extension`
-5. if there's a dialog with a warning, click on `Debug Anyway`
 
-By now, a new VS Code window should open up.  
-This is the window where you'll interact with your extension.  
-Whilst in the new window, open a git repo, which has any env0 environment associated with it  
+## 📋 Requirements
 
-*Note:* You will only see environments, in the organization of which you provided the api-keys   
+Before using this extension, make sure you meet the following prerequisites:
 
-That's it! any changes you make here would appear in the new window after you re-open / re-load the debug window (see below)
+1. **env0 Organization**: You must have an existing env0 organization.
+2. **API Credentials**: Provide the API Key ID and API Key Secret during login.
 
-## Some Tips
-instead of closing your debug window and reopening, hit:
-1. CMD+SHIFT+P
-2. search `reload`
-3. hit enter
+🔗 Learn how to [generate API keys](https://docs.env0.com/docs/api-keys).
 
-*Note* - some changes require to fully close the Debugging IDE (we've mainly encountered that when adding icons and such)
+## 🔑 Login and Logout
 
-Checking out the reading sources below is highly recommended
+To login or logout, simply use the `env0.login` and `env0.logout` commands through the VS Code command palette.
 
-## Extension Settings
+## 🚧 Limitations
 
-This extension contributes the following settings:
+**API Key & Organizations**: If you're using a personal API key and your account is associated with multiple organizations, the extension will default to a single organization and won't provide an option to select others. Use an Organization API key to resolve this.
```

Comment:
move `Use an Organization API key to resolve this.` to a separate line just so it'd be more visible